src/routes/sitemap.xml/+server.ts

```diff
@@ -0,0 +1,32 @@
+import { loadPageMetadataMap } from "$lib/loadPageMetadataMap";
+
+// https://kit.svelte.dev/docs/seo#manual-setup-sitemaps
+export async function GET() {
+  const { pathsToIDs } = await loadPageMetadataMap();
+  const urls = [...pathsToIDs]
+    .map(([path, _]) => `<url><loc>https://ldaf.la.gov${path}</loc></url>`)
+    .sort()
+    .join("");
+  // TODO: Include tags other than just <loc> within <url>, like:
+  //       * <lastmod> for date of last Contentful update
+  //       * <changefreq> for likeliness of page to change since last crawl;
+  //         we should probably default to 'weekly' but set some pages like
+  //         past events to 'never'
+  //       * <priority> for highlighting important pages; will require
+  //         updates to the content model
```

Comment:
Let me know if y'all think we should go ahead and add any of these now; I could see marking the `<changefreq>` as `"weekly"` for all URLs being a safe default.